json-rpc/src/stream_rpc/json_rpc.rs

```diff
@@ -31,6 +31,7 @@ impl CallableStreamMethod {
             StreamMethodRequest::SubscribeToEvents(params) => {
                 EventsSubscription::default().run(helper, params)
             }
+            StreamMethodRequest::Unsubscribe => unreachable!(),
```

Comment:
Why is this unreachable? might be worth a comment if we really aren't filling in the implementation here.